tests/integration/states/test_pkg.py

```diff
@@ -1059,6 +1059,7 @@ def test_pkg_cap_006_uptodate(self, grains=None):
             ret = self.run_state('pkg.removed', name=realpkg)
             self.assertSaltTrueReturn(ret)
 
+    @skipIf(True)
```

Comment:
Is this failing on other OS? It looks like it should only be  running on redhat/debian :suspect:

Reply:
Nope.  And it should be skipped, this was one of the spots where the tests were hanging.  The skipIf was just a test, not a permanent solution :smile:

Reply:
That's freaky. I don't see anything in here that should be causing anything to hang - grains.get maybe? You could always test that with a `start = time.time()` and then `if time.time() - start > THRESHOLD: assert False, 'Took too long'`, or something to that effect. I haven't been able to get reliable output on my tests without raising an exception.

Reply:
This test really would benefit from something like this though:

    @skipIf(not salt.utils.platform.is_darwin(), 'Test only available on macOS')